### PR TITLE
Quickfix for sorting "port_statuses" in TaskHeader.task_owner

### PR DESCRIPTION
**Pull request by mfranciszkiewicz**

### CODE REVIEW

File: golem/task/taskbase.py [hunk 1]
```diff
@@ -103,9 +103,16 @@ def dict_to_binary(cls, dictionary):
         self_dict.pop('last_checking', None)
         self_dict.pop('signature', None)
 
+        port_statuses = self_dict['task_owner'].get('port_statuses')
```
Comment (maaktweluit) [inline golem/task/taskbase.py]: Maybe add a comment on why ordering is supposed to work to fix later?

**Review by maaktweluit — APPROVED** [review]
LGTM!

File: golem/task/taskbase.py (hunk 1 above)
Comment (mfranciszkiewicz) [reply golem/task/taskbase.py]: Done

**Review by jiivan — APPROVED** [review]
:+1:

**Pull request merged**